headless-server-commerce-parent/pom.xml

```diff
@@ -47,10 +47,10 @@
     <!-- versions managed in spring-boot-starter-parent (overwrites) -->
     <graphql-java.version>19.9</graphql-java.version>
     <netty.version>4.1.106.Final</netty.version>
-    <maven-enforcer-plugin.version>3.4.1</maven-enforcer-plugin.version>
+    <maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
```

Comment:
Although I recall that we did this together I'd like to know if there's a newer enforcer plugin that we can use here. At the very least we should have a CMS ticket reminding us that we'll have to update the enforcer version in CMS workspace and adapt the enforcer rules accordingly.